tools/SRS_tools/biodiv_indices_global.xml

```diff
@@ -0,0 +1,116 @@
+<tool id="SRS_global_indices" name="Compute biodiversity indices" version="@VERSION@" profile = "20.01">
+    <description>from remote sensing data</description>
+    <macros>
+        <import>macro.xml</import>
+    </macros>
+    <requirements>
+        <requirement type="package" version="3.5_15">r-raster</requirement>
+        <requirement type="package" version="1.5_29">r-rgdal</requirement>
+        <requirement type="package" version="1.4_7">r-sp</requirement>
+        <requirement type="package" version="0.2_5.2">r-rasterdiv</requirement>
+        <requirement type="package" version="3.3.6">r-ggplot2</requirement> 
+    </requirements>
+    <command detect_errors="exit_code"><![CDATA[ 
+        #if $method.type == 'envi_bil': 
+          cp '$input_raster' '${input_raster.element_identifier}' &&
+          cp '$input_header' '${input_header.element_identifier}' &&
+        #end if 
+        Rscript
+            '$__tool_directory__/biodiv_indices_global.r'
+            #if $method.type == 'envi_bil':
+              '$method.input_raster.element_identifier'
+              '$method.input_header.element_identifier'
+              ''
+            #else:
+              ''
+              ''
+              '$method.input'
+            #end if
+            '$alpha'
+            '$__tool_directory__/functions.r'
+            '$output_indices'
+            '$plots_png'
+        ]]>
+    </command>
+    <inputs>
+        <conditional name="method">
+            <param name="type" type="select" label="In which format are your data ?">
+                <option value="zipper">The data you are using are in a zip folder Reflectance</option>
+                <option value="envi_bil">Your already have the files in ENVI BIL format</option>
+            </param>
+            <when value="zipper">
+                <param name="input" type="data" format="data" multiple="true" label="Input data"/>
```

Comment:
format=data is too unspecific. It should be a valid filetype

Reply:
Will do ! thank you again !!